variables.tf

```diff
@@ -128,6 +128,12 @@ variable "enable_azure_policy" {
   default     = false
 }
 
+variable "enable_open_service_mesh" {
+  description = "Enable Open Service Mesh Addon."
+  type        = bool
+  default     = false
```

Comment:
Would the default value `null` be better? A `false` default value will cause changes in plan when the users upgrade module's value.